code/game/machinery/doors/airlock.dm

```diff
@@ -2095,12 +2095,12 @@
 	glass = TRUE
 //////////////////////////////////
 /*
-	Station2 Airlocks
+	Public Airlocks
```

Comment:
If you're touching these please just replace them with 
`// Public Airlocks`

The "fancy" header fucks up our jsdocs and also takes up space for little benefit.

Reply:
Sure